Makefile

```diff
@@ -7,30 +7,13 @@ $(VIRTUALENV)/.installed:
 	@if [ -d $(VIRTUALENV) ]; then rm -rf $(VIRTUALENV); fi
 	@mkdir -p $(VIRTUALENV)
 	virtualenv --python $(PYTHON_VERSION) $(VIRTUALENV)
-	$(VIRTUALENV)/bin/pip3 install -r requirements.txt
 	$(VIRTUALENV)/bin/pip3 install -r requirements_test.txt
-	$(VIRTUALENV)/bin/pip3 install -e .
-	$(VIRTUALENV)/bin/pip3 install git+https://github.com/epfml/sent2vec.git # otherwise it fails cause setup.py relies on cython and numpy being installed
```

Comment:
I think the problem @aCampello was caused by removing this line not requirements. Testing that theory now.

Reply:
I think the problem is that when we download sent2vec it doesn't install the dependencies of that project.

Reply:
Issue #83 and resolved by #84

Reply:
👍